src/components/PostInfo/PostInfo.jsx

```diff
@@ -2,0 +1,27 @@
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} key={post.id} />
+      </p>
+    </div>
+
+    <p className="PostInfo__body">{post.body}</p>
+
+    <hr />
+
+    {post.comments.length >= 1 ? (
+      <CommentList comments={post.comments} key={post.id} />
```

Comment:
The `key` prop is not needed for the `CommentList` component here. The `key` prop is typically used when rendering lists of elements to help React identify which items have changed, are added, or are removed.